### PR TITLE
fix(components/lookup): use `position: absolute` for autocomplete dropdown

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**
[AB#2480553](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2480553)

*Mobile Safari, position: fixed and the virtual keyboard — an erroneous combination*
https://medium.com/@im_rahul/safari-and-position-fixed-978122be5f29

**StackOverflow article**
https://stackoverflow.com/questions/43833049/how-to-make-fixed-content-go-above-ios-keyboard

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1176/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1176/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1176/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1176/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1176/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1176/playground/)

**Review by johnhwhite — DISMISSED** [review]

File: libs/components/core/src/lib/modules/affix/affix-config.ts [hunk 1]
```diff
@@ -45,4 +46,9 @@ export interface SkyAffixConfig {
    * The vertical alignment of the affixed element to the base element.
    */
   verticalAlignment?: SkyAffixVerticalAlignment;
+
+  /**
+   * The position of the element being fixed.
```
Comment (Blackbaud-SteveBrush) [inline libs/components/core/src/lib/modules/affix/affix-config.ts]: Should be `being affixed.`?

File: libs/components/core/src/lib/modules/affix/affixer.ts [hunk 2]
```diff
@@ -233,13 +246,33 @@ export class SkyAffixer {
...
+      bottom: baseDomRect.bottom,
+      left: baseDomRect.left,
+      right: baseDomRect.right,
+      width: baseDomRect.width,
+      height: baseDomRect.height,
+    };
+
+    if (this.#config.position === 'absolute') {
+      baseRect.top += window.scrollY;
+      baseRect.bottom = baseRect.top + baseDomRect.height;
```
Comment (Blackbaud-SteveBrush) [inline libs/components/core/src/lib/modules/affix/affixer.ts:263]: Do your calculations also account for a horizontal scroll?

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See null

**Pull request merged**